CHANGELOG.md

```diff
@@ -24,12 +24,6 @@ Changes are grouped as follows
 ### Added
 - `DetectJobBundle` dataclass: A way to manage multiple files and jobs.
 
-## [4.11.2] - 2022-11-16
-### Changed
-- Detect endpoint (for Engineering Diagram detect jobs) is updated to spawn and handle multiple jobs.
-### Added
-- `DetectJobBundle` dataclass: A way to manage multiple files and jobs.
```

Comment:
was 4.11.12 never released and also its changes not included into 4.11.3?

Reply:
It was released. Just removing because it's duplicated.